src/components/Layout/SymptomsCards/index.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import './SymptomsCards.scss';
+import '../../../styles/_colors.scss'
+import Tiredeness from '../../../assets/images/tiredness.svg';
+import Fever from '../../../assets/images/fever.svg';
+import ShortnessBreath from '../../../assets/images/shortness-breath.svg';
+import Diarrhoea from '../../../assets/images/diarrhoea.svg';
+import Cough from '../../../assets/images/cough.svg';
+import RunnyNose from '../../../assets/images/runny-nose.svg';
+const Cards = () => {
+    return (
+        <React.Fragment>
+          <section className='Carrousel'>
+            <h4 className='Carrousel__title'>Syptoms</h4>
+            <div className='Carrousel__container'>
+              <div className='Carrousel__item'>
+                <div className='Carrousel__item--image'>
+                  <img src={Tiredeness} alt="Avatar"/>
+                </div>
+                <div className='Carrousel__information'>
+                  <h4 className='Carrousel__title'>Tiredeness</h4>
+                  <p className='Carrousel__description'> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt</p>
+                </div>
+              </div>
```

Comment:
Hello @Sergioleal7 
I think this can be turned into a component and could pass the image and text via props.

Reply:
Yes @Sergioleal7  the idea is to create all the components UI, you are repeating many HTML

Reply:
Toda la razón. Estoy trabajando en ese cambio. Lo subo ahora @georgehossa @wolftrax5